thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/resolution/component.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Filter Bar Component
```

Comment:
The jsDoc here doesn't seem to match the component.

Reply:
done